_nds/TWiLightMenu/3dsmenu/themes/meta/TLoZ/info.json

```
@@ -0,0 +1,7 @@
+{
+	"title": "The Legend of Zelda",
+	"author": "Allinxter",
+	"description": "Blue, green and golden colors, a landscape of Hyrule and a familiar tune welcome you to TWiLight Menu++",
+	"categories": ["Legend of Zelda", "Zelda", "BoTW", "Link", "Green"],

```

```suggestion
	"categories": ["legend-of-zelda", "zelda", "botw", "link", "green"],
```
The categories should be all lowercase and no spaces, otherwise looks good